kubernetes-manifests/service.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: restful-distributed-lock-manager
+    group: com.stakater.platform
+    version: 0.5.1
+    provider: stakater
+  name: app
```

Comment:
Shouldn't we name the deployment and service to like "restful-distributed-lock-manager"?

Reply:
good catch!